nntrainer/layers/loss_layer.cpp

```diff
@@ -128,15 +128,31 @@ void LossLayer::calcDerivative() {
 
   switch (loss_type) {
   case LossType::LOSS_MSE:
-    ret_derivative = y.subtract(y2).multiply(2).divide(y.getDim().getDataLen());
+    y.subtract(y2, ret_derivative);
+    ret_derivative.multiply_i(2);
+    if (ret_derivative.divide_i(y.length()) != ML_ERROR_NONE) {
```

Comment:
Please do not review yet, this patch is meant for the testing purpose as of now, until the underlying bug is fixed.

Reply:
I guess this is ready to review now?

Reply:
Yup